flume-ng-sinks/flume-ng-kudu-sink/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
nit: I think (at least for consistency) this should be a log4j2.xml file like https://github.com/apache/flume/blob/trunk/flume-ng-sinks/flume-hdfs-sink/src/test/resources/log4j2.xml